js/foundation.sticky.js

```diff
@@ -46,7 +46,34 @@ class Sticky {
         $('#' + _this.options.anchor).attr({ 'data-mutate': id });
     }
 
+    // Needs refactoring
+    function translateFix() {
+      var topOffset = $(window).scrollTop();
+
+      _this.$element.css('transform','translateY(' + topOffset + 'px)');
+    }
+
```

Comment:
the problem is binding - by default callback will have window binding so this won't be there. You can address this by forcing _translateFix to have `this` of the object, for example by defining it using `=>` during `_init`.  For example see https://github.com/zurb/foundation-sites/blob/develop/js/foundation.accordion.js#L66